App.tsx

```diff
@@ -57,7 +55,7 @@ function GeolocationSwitch() {
         StorageKeys.ShouldBeTrackingFlagStorageAdress,
       );
       if (value !== undefined && value !== null) {
-        if (value == 'true') {
+        if (value == 'true' || value == '"true"') {
```

Comment:
Is this double check really useful?

Reply:
I am not 100% sure. But it is not risky and I want to avoid breaking the CozyGPSMemory UI.

Previous AsyncStorage setter and getter was working only with string. New one parse in JSON and vice versa automatically. In some tests, I think when the two are working together (like here in App.tsx) it may leads to store `"true"` and get it as is.